.github/workflows/configs-json-lint.yml

```diff
@@ -0,0 +1,26 @@
+name: configs-json-lint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: configs JSON lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check configs JSON format
+        run: |
+          files=("config_example.json" "testdata/configtest.json")
```

Comment:
Suggestion: might be able to use `files=$(git ls-files '*.json')` so we can conform everything and for future files and filter by config key word. But that seems like a pain. So only a suggestion. 

here's the output:

<img width="564" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/b9c15f90-913f-407f-a30c-5265fdf8cf3f">

Reply:
Yep, it was made to be extensible but this PR only targets our mainly touched configs for now